src/cookie.js

```diff
@@ -1,48 +1,7 @@
 import * as utils from './utils';
 import adLoader from './adloader';
-import { StorageManager, pbjsSyncsKey } from './storagemanager';
 
 const cookie = exports;
-const queue = [];
-
-function fireSyncs() {
-  queue.forEach(obj => {
-    utils.logMessage(`Invoking cookie sync for bidder: ${obj.bidder}`);
-    if (obj.type === 'iframe') {
-      utils.insertCookieSyncIframe(obj.url, false);
-    } else {
-      utils.insertPixel(obj.url);
-    }
-    setBidderSynced(obj.bidder);
-  });
-  // empty queue.
-  queue.length = 0;
-}
-
-function setBidderSynced(bidder) {
```

Comment:
I'd like to preserve this functionality if we can. How about a callback on the pixel queue?